src/pages/workspaces/workspace/analysis/modals/persistent-disk-controls.ts

```diff
@@ -120,48 +140,124 @@ export const PersistentDiskSection = (props: PersistentDiskControlProps) => {
           ['Learn more about persistent disks and where your disk is mounted.']
```

Comment:
This should have a popout icon if it opens something outside of Terra (similar to the learn more links above).